3-networks-dual-svpc/common.auto.mod.tfvars

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// The DNS name of peering managed zone. Must end with a period.
+domain = "DOMAIN.REPLACE.ME." # REPLACE ME (manually)
+
+// Update the following line and add you email in the perimeter_additional_members list.
+// You must be in this list to be able to view/access resources in the project protected by the VPC service controls.
+
+perimeter_additional_members = ["user:USER_ID@DOMAIN."] # REPLACE ME (manually)
```

Comment:
removed trailing . in #421 
Tracking for full verification in an org level test in #422